httpsupgrade/httpsupgrade-impl/src/main/java/com/duckduckgo/httpsupgrade/impl/HttpsUpgrader.kt

```diff
@@ -99,11 +99,8 @@ class HttpsUpgraderImpl @Inject constructor(
     override fun reloadData() {
         logcat { "Reload Https upgrader data" }
         bloomReloadLock.lock()
-        try {
-            bloomFilter = bloomFactory.create()
-        } finally {
-            bloomReloadLock.unlock()
-        }
+        runCatching { bloomFactory.create() }.getOrNull()
```

Comment:
Don’t we need to assign this to bloomFilter @aitorvs

Reply:
🤦  done